external_services/cache.py

```diff
@@ -49,6 +50,7 @@ def append_to_group(groups, group_order, group_label, menu_entry):
         }
 
     def student_link_groups(self):
+        # make sure the items with empty group
         return self.data['student_groups']
```

Comment:
I believe this comment was forgotten here and should be removed.

Reply:
Yes